pkg/frr/configure.go

```diff
@@ -91,15 +93,26 @@ func (f *FRRManager) renderSubtemplates(in FRRConfiguration) (*FRRTemplateConfig
 	if err != nil {
 		return nil, err
 	}
+	envECMPMaximumPaths, isSet := os.LookupEnv("NWOP_ECMP_MAXIMUM_PATHS")
+	ecmpMaximumPaths := 8 // Default to BCMs preferred target of maximum 8 paths per route.
+	if isSet {
+		parsedECMPMaximumPaths, err := strconv.Atoi(envECMPMaximumPaths)
+		if err != nil {
+			ecmpMaximumPaths = parsedECMPMaximumPaths
+		} else {
+			return nil, err
+		}
+	}
```

Comment:
I would implement it like this, which is a more common style and reduces scope of some of the variables.

In addition the if condition for error handling was the wrong way round, which is corrected here as well.

```suggestion
	ecmpMaximumPaths := 8 // Default to BCMs preferred target of maximum 8 paths per route.
	if envECMPMaximumPaths, ok := os.LookupEnv("NWOP_ECMP_MAXIMUM_PATHS"); ok {
		parsedECMPMaximumPaths, err := strconv.Atoi(envECMPMaximumPaths)
		if err != nil {
			return nil, err
		}
		ecmpMaximumPaths = parsedECMPMaximumPaths
	}
```